app/src/main/java/ru/androidacademy/gooseeye/models/MovieInfo.kt

```diff
@@ -0,0 +1,10 @@
+package ru.androidacademy.gooseeye.models
+
+data class MovieInfo(var moviePoster: Int,
+                     var age: String,
+                     var tag: String,
+                     var reviews: String,
+                     var movieName: String,
+                     var duration: String,
+                     var rating: Float,
+                     var like: Boolean)
```

Comment:
If you don't change the field use val.

Reply:
it is good practice to keep all fields of data class immutable